Engine/ScriptAnalyzer.cs

```diff
@@ -1890,6 +1853,51 @@ private IEnumerable<DiagnosticRecord> AnalyzeFile(string filePath)
                 return null;
             }
 
+            // short-circuited processing for a help file
+            // no parsing can really be done, but there are other rules to run (specifically encoding).
+            if ( Regex.Matches(Path.GetFileName(filePath), @"^about_.*help.txt$", RegexOptions.IgnoreCase).Count != 0)
```

Comment:
This regex might benefit from being a compiled `private readonly static` field:

```csharp
private readonly Regex s_aboutHelpRegex = new Regex("^about_.*help.txt$", RegexOptions.IgnoreCase | RegexOptions.Compiled);
```

Then here:
```suggestion
            if (s_aboutHelpRegex.IsMatch(Path.GetFileName(filePath)))
```

Reply:
Or the other possibility is just to do:

```csharp
string fileName = Path.GetFileName(filePath);
if (fileName.StartsWith("about_", StringComparison.OrdinalIgnoreCase)
    && fileName.EndsWith("help.txt", StringComparison.OrdinalIgnoreCase))
{
...
}
```

Reply:
I went with the regex